### PR TITLE
Adds mention of photography in policies

### PR DESCRIPTION
**Pull request by bdamerac**
Addresses #211 .

### CODE REVIEW

File: content/pages/photography-policy.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+We often take pictures at events, but not without consent.
```
Comment (evykassirer) [inline content/pages/photography-policy.md:9]: this needs meta data at the top (content page, status hidden, etc)

**Comment by evykassirer** [comment]
yay deploying works <3

File: content/pages/photography-policy.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+Title: Photography Policy Category: pages Status: hidden Slug: photography-policy
```
Comment (evykassirer) [inline content/pages/photography-policy.md]: this has to be on separate lines - see how it breaks in the review app

Comment (bdamerac) [inline content/pages/photography-policy.md]: will fix that now

**Comment by evykassirer** [comment]
also noting that this is a continuation of #233

File: content/pages/photography-policy.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+Title: Photography Policy
+Category: pages
+Status: hidden
+Slug: photography-policy
+
+We often take pictures at events, but not without consent.
+We will not take you picture if you're wearing a lanyard (We keep
```
Comment (evykassirer) [inline content/pages/photography-policy.md]: "you"->"your"

Comment (evykassirer) [inline content/pages/photography-policy.md]: and lowercase w for we I think

File: content/pages/photography-policy.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+We can use these pictures for our social media profiles (Facebook,
+Twitter etc.) and also for our publicity material like brochures.
+
+We always tell everyone at the beginning of the event that photos
+taken will end up on social media unless you specifically ask otherwise
+or are wearing a lanyard.
+
+If you have any questions about photography, email us at
+[wics-ugrad@lists.uwaterloo.ca](mailto:wics-ugrad@lists.uwaterloo.ca)
```
Comment (evykassirer) [inline content/pages/photography-policy.md]: period at the end of this sentence

File: content/pages/photography-policy.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+We often take pictures at events, but not without consent.
+We will not take you picture if you're wearing a lanyard (We keep
+some lanyards available at all our events) or if you request that we
+don't take your picture.
+
+We can use these pictures for our social media profiles (Facebook,
+Twitter etc.) and also for our publicity material like brochures.
+
+We always tell everyone at the beginning of the event that photos
+taken will end up on social media unless you specifically ask otherwise
```
Comment (annalorimer) [inline content/pages/photography-policy.md]: "end up" -> "will be posted to"

File: content/pages/photography-policy.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+We often take pictures at events, but not without consent.
+We will not take you picture if you're wearing a lanyard (We keep
+some lanyards available at all our events) or if you request that we
+don't take your picture.
+
+We can use these pictures for our social media profiles (Facebook,
+Twitter etc.) and also for our publicity material like brochures.
+
+We always tell everyone at the beginning of the event that photos
```
Comment (annalorimer) [inline content/pages/photography-policy.md]: "We will always tell" -> "We will review the photography policy at the beginning of every event we run"

**Comment by annalorimer** [comment]
What do people think about adding section headers? i.e "Enforcing the Photography Policy" would detail how the lanyards are used, "How Photographs Are Used" would detail what we use the photos for. 

Just thinking that dividing up the sections might make it clearer.

**Comment by evykassirer** [comment]
that seems reasonable!

**Comment by evykassirer** [comment]
yay looks good thanks for working on this so quickly @Bhargavi1997 !

And thanks for taking a look @annalorimer and @arshiamufti 💕

**Pull request merged**